Automattic/wp-calypso
# Help center: Update contact us options

Related to #

## Proposed Changes

* This PR modifies the method by which our users interact with us through the Help Center. Now, by default, paid users will have live chat enabled, which has been renamed to 'Contact WordPress.com Support.' Additionally, there is a toggle option allowing users to enable contact via email. Please apply these changes before. D148058-code
If the toggle is 'On' users get only the email contact, otherwise, 'Contact WordPress.com Support.'

* Furthermore, the forums option button was removed.
* Item 3 and 6 from this pdDR7T-1vN-p2#comment-1926 will be addressed in another further PR.

For better understanding, please check pdDR7T-1vN-p2

## Testing Instructions



* Apply this diff D148058-code
* Pull and run this branch
* Follow the steps on the diff if you haven't (toggle on or off the email contact)
* Run `yarn dev --sync` to sync ETK
* Navigate to a sandboxed simple site of yours
* Open help center and click on “Still need help?”
* You will get the “Wapuu” AI bot, just type “hello, this is a test” and submit
* Next, rate the feedback as negative
* You should see the contact options below, either email (if the toggle is on) or “Contact WordPress.com Support” if the toggle is off. By default, it is off.

Check also on atomic, remember to compress and update ETK to your atomic site, and make sure you add `define('JETPACK__SANDBOX_DOMAIN', 'you_sandbox_name_url');` in your wp-config.php to be able to reach the endpoint in your sandbox (of course after applying the diff D148058-code)

## Pre-merge Checklist
